sumologic/resource_sumologic_cse_inventory_entity_group_configuration.go

```diff
@@ -41,6 +42,23 @@ func resourceSumologicCSEInventoryEntityGroupConfiguration() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"inventory_key": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"inventory_value": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"tag_schema": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
Do these new fields need a an empty string default `Default: ""`? Other than that, looks good to me.

Reply:
@kachristensen , the new fields doesn't not have a default value - the one with default is the deprecated field `group`